pages/discover.js

```diff
@@ -70,18 +72,17 @@ export default function Popular({ data, BASE_URL }) {
         title="Discover LinkFree Profiles"
         description="Discover more people in your LinkFree community"
       />
-
       <Page>
         <h1 className="text-2xl md:text-4xl mb-4 font-bold">
           Discover LinkFree Profiles
         </h1>
 
         <div className="flex flex-wrap justify-center mb-4">
-          {data.tags &&
-            data.tags
+          {tags &&
+            tags
               .slice(0, 10)
               .map((tag) => (
-                <Tag name={tag.name} key={tag.name} total={tag.total} />
+                <Tag name={tag.name} key={tag.name} total={tag.total} path='/discover'/>
```

Comment:
I get an error when clicking on a tag from the Discover page

<img width="1680" alt="Screenshot 2023-02-01 at 15 15 52" src="https://user-images.githubusercontent.com/624760/215987909-f61019e8-026c-43ef-9ab4-51525ced24fe.png">

Reply:
Yeah I will write the condition for undefined

Reply:
If there are any other issues that you noticed do let me know

Reply:
I think this is the last one, thank you for your persistence 💪 hopefully we will get this merged very soon 🎉

Reply:
Great! I will test everything one last time and make the commit soon.

Reply:
@eddiejaoude made the last commit with all fixes needed.